src/ip_grid_mod.f90

```diff
@@ -1,25 +1,60 @@
+!> @file
+!! @brief Abstract ip_grid type
+!!
+!! @author Kyle Gerheiser
+!! @date July 2021
+
+!> Abstract ip_grid type
 module ip_grid_mod
   use ip_grid_descriptor_mod
   implicit none
 
+  !> @param Integer grid number for equidistant cylindrical grid in grib1
```

Comment:
We usually don't use the @param tag for module variables. What does it do? The variable is not a parameter, right?

Normally we would document these like:
  `integer, public, parameter :: EQUID_CYLIND_GRID_ID_GRIB1 = 0 !< grid number for equidistant cylindrical grid in grib1`

Reply:
It's a way to mark module variables. Doxygen does the correct thing, and lists it under module variables.

I like it more than the other way because it's on a separate line.

![Screen Shot 2021-07-29 at 10 34 28](https://user-images.githubusercontent.com/3209794/127511311-e7cd9c57-80f4-4871-82e5-ce1a60b75215.png)

Reply:
That actually wasn't doing what I thought it was. Fixed.